BuildGuide.md

```diff
@@ -0,0 +1,126 @@
+
+Keyboard testing websites
+
+* [Keyboard Checker](https://keyboardchecker.com/)
+* [Keyboard Tester](https://www.keyboardtester.com/)
+
+Quick Tip: Instead of looking up after testing each key, it is faster to test every key in a row, then ensure that the entire row lights up.
+
+Note: You will not see the function key light up via this method. You will need to use a second pair of tweezers in conjunction to ensure that it registers. Our favorite is FN + 1, which will open a new tab when in chrome.
+
+## Dissasembly
+
+Remove the board from the case and use the included cloth to lay the board face down on a flat surface. You will need to remove the eight hex screws (blue underline below) from the back of the board and gently lift the base up, and pull the base out an angle towards the back of the board.
+
+![iron_back](Photos/Iron180Photos/iron_back_mod.jpg)
+![remove back](Photos/Iron180Photos/remove_back_mod.jpg)
+
+## Bumpons, Daughterboard, and Gaskets
+
+Since we are going to move halves of the boards around individually, it is now a good time to attach the bumpons. Remove a bumpon from the adhesive, then attach it to the bottom of the base (red underline below) using one of the four circular matching indents as a guide. Repeat for the other three bumpons.
+
+![bumpon](Photos/Iron180Photos/bumpon_mod.jpg)
+
+Proceed to flip the board face-up, grab the four smaller screws and attach the daughterboard to the case using the four smallest screws (orange underline in parts and diagram below).
+
+Note: Do not overtighten the screws; turn them just enough to where it starts to give resistance, and the daughterboard no longer jiggles. It is also easier if you attach one end of the cable to the daughterboard before screwing it in. If you happen to install the gaskets prior to installing the daughterboard, you should still be able to access the top screws without disturbing the gasket adhesive.
+
+![daughterboard](Photos/Iron180Photos/daughterboard_mod.jpg)
+![attached daughterboard](Photos/Iron180Photos/attached_daughterboard.jpg)
+
+Now that your bumpons and daughterboard are attached, we will focus on attaching the gaskets.
+
+We recommend you use a pair of tweezers for this step, but you can get by just using your hands. You will notice a slight recess on the inside of both the top and bottom of the 180 which we will use as a guide for gasket application. While removing adhesive protection and attaching gaskets to a board sounds trivial, we do have a few tips for quality control.
```

Comment:
`get by with`

Reply:
`on both the insides of the top and bottom`

Reply:
`... attaching gaskets to a board *may* sound`

Reply:
integrated all changes